content/pages/mua-summary.md

```diff
@@ -0,0 +1,68 @@
+Title: MUA Summary
+Category: pages
+Status: hidden
+Slug: mua-summary
+
+Anyone with a user account to a WiCS machine must sign our [Machine Usage Agreement]({filename}/pages/mua.md),
+summarized below. Those with guest accounts do not need to sign the MUA, but they are still held
+to the rules and regulations outlined in it.
+
+# Acceptable and Unacceptable Use #
+
+The WiCS machines are to be used for learning, class work, personal projects, and 
+use in accordance with [WiCS aims]({filename}/pages/about.md), that does not violate
+any of WiCS’ usage policies.
+
+If you have a project of interest that does not fall into these categories, you may
+request special priority from wics-sys.
+
+# User Responsibilities #
+
+All usage on WiCS machines is governed by our [CoC]({filename}/pages/code-of-conduct.md), 
+our [MUA]({filename}/pages/mua.md), local laws, and University policy. Users are also
+responsible for being aware of and adhering to policy of machines connected to from 
+WiCS machines.
+
+WiCS is not responsible for any activity on user accounts. Users will be held accountable 
+for all damaging, unethical and illegal activity.
+
+Users should be considerate of others’ needs when using WiCS machines. Do not disrupt the 
+work of anyone else, and do not hog resources.
+
+# Security #
+
+Users are only allowed to access their own accounts. Do not attempt to gain access to another’s
+account or personal file. 
+
+Do not attempt to obtain anyone else’s password, and do not give your password or access to your account to anyone else. 
+
+Choose a password that you do not use anywhere else, and do not give an machine automatic access
+to your WiCS account (via .rhosts files etc.)
+
+In the event that a security problem is found, members of [wics-sys](mailto:wics-sys@lists.uwaterloo.ca)
+must be notified immediately. The problem should not be exploited, and it should not be made
+public until it is corrected.
+
+# Auditing and Logging #
+
+Wics-sys will keep logs on all machines for diagnostic and auditing purposes. On all machines
+these are located under /var/logs.
+
+# Rights of the Systems Committee and the WiCS Executive #
+
+Wics-sys may examine any file or program that appears to be in violation of usage policies.
+These processes may be killed, stopped, or interrupted at the discretion of the Systems
+Committee, and he owner of the process will be notified.
+
+The Systems Committee may revoke a user’s access to an account provided that written 
+explanation is given. Any violation of the MUA, CoC, University policy, or local law 
+may result in removal of access. Users may appeal to the WiCS committee for reinstatement
```

Comment:
Previously, we've used WiCS committee to mean "big WiCS". I'd write out the full and proper form of which committee we're talking about here.